updater/lib/dependabot/base_command.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require "raven"
+require "sentry-ruby"
```

Comment:
I'm unclear if we actually need this given we already `require "sentry-ruby"` in  `fetch_files.rb`/`update_files.rb`.